# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  73013e362352a830d5d2f35325403166e1af75f2
Misc: CSS tweak so download link aligns correctly.

diff --git a/mail/base/content/aboutDialog.css b/mail/base/content/aboutDialog.css
--- a/mail/base/content/aboutDialog.css
+++ b/mail/base/content/aboutDialog.css
@@ -132,10 +132,11 @@
   padding: 0;
   font-weight: bold;
 }
 
 #noUpdatesFound label {
   margin-inline-start: 0;
 }
 #manualUpdate label {
-  margin-inline-start: 0;
+  margin-inline: 0 0.3em;
+  margin-block: 1px 2px;  /* align with link */
 }
